app/templates/gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules<% if (nyc) { %>
 .nyc_output
 coverage<% } %>
+yarn.lock
```

Comment:
I like neither `yarn` nor `npm`. It's just that I don't use `yarn` as it's been too unstable for me. I guess you're right though, should be included to prevent it accidentally included in PRs.